API.md

```diff
@@ -211,34 +210,6 @@ Add `PackageProviders().Append(winrt::ReactNativeVideoCPP::ReactPackageProvider(
 
 Follow the manual linking instuctions for React Native Windows 0.62 above, but substitute _ReactNativeVideoCPP61_ for _ReactNativeVideoCPP_.
 
-</details>
-
```

Comment:
I think this may have been removed by accident. A huge chunk of the document is now hidden inside the `Windows RNW C++/WinRT details`  dropdown

Reply:
I'll take a look. Thanks!